PubNubUnity/Assets/PubNub/PubNubUnity/PNConfiguration.cs

```diff
@@ -26,23 +28,29 @@ public PNConfiguration ()
         public string PublishKey {get; set;}
         public string SecretKey {get; set;}
         public string CipherKey {get; set;}
-        private string uuid;
-        public string UUID { 
+        private string userId;
+        public string UserId { 
             get{
-                if (string.IsNullOrEmpty (uuid) || string.IsNullOrEmpty (uuid.Trim ())) {
-                    uuid = string.Format("pn-{0}", Guid.NewGuid ().ToString ());
+                if (string.IsNullOrEmpty (userId) || string.IsNullOrEmpty (userId.Trim ())) {
+                    userId = string.Format("pn-{0}", Guid.NewGuid ().ToString ());
                 }
 
-                return uuid;
+                return userId;
             }
             set{
-                uuid = value;
-                if(UUIDChanged!=null){
-                    UUIDChanged.Invoke(this, null);
-                }
+                userId = value;
+                UUIDChanged?.Invoke(this, null);
+                UserIdChanged?.Invoke(this, null);
```

Comment:
Two invokes? I guess second one redundant. Just checking if all is good with the two invokes. Please confirm.

Reply:
Both events are still present - once we fully commit to removing UUID, this invoke will be deleted as well. I guess I could just subscribe one event to the other, but that would be mostly equivalent.

Reply:
On second thought - I made it more consistent, so the double invocation has been removed.